src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/LanguageServerConstants.cs

```diff
@@ -9,13 +9,6 @@ public static class LanguageServerConstants
 
         public const string ProjectConfigurationFile = "project.razor.json";
 
-        // Semantic "Legacy" endpoints refer to an old LSP spec version, needed for now until VS reacts.
```

Comment:
@ryanbrandenburg could you review all the semantic token pieces of this PR. I ran everything and validated that semantic tokens light up in VS now that they've migrated as part of this refactor but want to make sure I'm not missing anything.